test/csuite/random_directio/main.c

```diff
@@ -806,9 +806,9 @@ check_db(uint32_t nth, uint32_t datasize, pid_t pid, bool directio, uint32_t fla
         large_arr[th] = dcalloc(LARGE_WRITE_SIZE, 1);
         large_buf(large_arr[th], LARGE_WRITE_SIZE, th, true);
     }
-    testutil_check(__wt_snprintf(checkdir, sizeof(checkdir), "%s.CHECK", home));
-    testutil_check(__wt_snprintf(dbgdir, sizeof(savedir), "%s.DEBUG", home));
-    testutil_check(__wt_snprintf(savedir, sizeof(savedir), "%s.SAVE", home));
+    testutil_check(__wt_snprintf(checkdir, sizeof(checkdir), "../%s.CHECK", home));
+    testutil_check(__wt_snprintf(dbgdir, sizeof(savedir), "../%s.DEBUG", home));
+    testutil_check(__wt_snprintf(savedir, sizeof(savedir), "../%s.SAVE", home));
```

Comment:
`.SAVE` folders are also created in `testutil_copy_data`, where they're placed in the parent directory. 
I've updated the path of `.SAVE` to match this, and the paths of `.CHECK`, `.DEBUG`, `.BACKUP` for consistency